core/src/test/scala/anorm/ColumnSpec.scala

```diff
@@ -156,6 +156,18 @@ object ColumnSpec
       SQL("SELECT i").as(scalar[Long].single) aka "parsed long" must_== 4l
     }
 
+    "be parsed from short" in withQueryResult(shortList :+ 3.toShort) {
+      implicit con =>
+        SQL("SELECT s").as(scalar[Long].single).
+          aka("parsed short") must_== 3l
```

Comment:
Sorry I just followed the convention of the file... (it's lower case everywhere else)